database/gui_repo_worker.py

```
@@ -10,14 +10,15 @@
 
 load_dotenv()
 ROOT_DIR = Path(os.getenv("ROOT_DIR") or "")
+DB_PATH = Path(os.getenv("DB_PATH") or "comics.db")
 
 
 class RepoWorker:
     def __init__(self):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)